### ISSUE TITLE
Dynamic shoot stick

### ISSUE THREAD

**Issue by Thanadrax** [aarcangeli/Serious-Sam-Android#10]
Hi! First of all huge thanks for making such a diamond. You rock. 
So, my suggestion is:

Most of shooters on android have the shoot stick which makes aiming while shooting possible. It would be really cool to see it in android port.

**Comment by aarcangeli** [comment]
> First of all huge thanks for making such a diamond. You rock.

Thank you very mutch. I appreciate it

> Most of shooters on android have the shoot stick which makes aiming while shooting possible. It would be really cool to see it in android port.

With "makes aiming" do you mean move the view automatically so that you can target an enemy?
I don't know any games like that, can you give me an example?

**Comment by Thanadrax** [comment]
> > First of all huge thanks for making such a diamond. You rock.
> 
> Thank you very mutch. I appreciate it
> 
> > Most of shooters on android have the shoot stick which makes aiming while shooting possible. It would be really cool to see it in android port.
> 
> With "makes aiming" do you mean move the view automatically so that you can target an enemy?
> I don't know any games like that, can you give me an example?

No, what i mean is that some games have movable shoot stick. Just like walking stick, but this one is responsible for moving camera while shooting. 
These games have it:
Delta Touch
PUBG Mobile
And so on.
Uhh.. i don't know how to explain it properly, so i will describe it straightly: It would be good if player was able to move the camera while shooting.

**Comment by aarcangeli** [comment]
It would be really good if player could be able to move the camera while he shoots.
But you can use up to two fingers at the same time.

If I put a second joystick (on the right), you will be going to move the camera, but you will not be able to shoot anyway.
I don't know in other games, but in Serious Sam the perfect timing is crucial.
You must shoot exactly when the enemy is under the target, you can't just push the button and move the view at the same time.

If you want to try, you can install version [1.02.1](https://github.com/aarcangeli/Serious-Sam-Android/releases/tag/v1.02.1) (note that you must uninstall the current version in order to downgrade)

In conclusion I **can** put a second joystick but it will not solve the problem.

**Comment by xcoress** [comment]
@aarcangeli I'm sure everyone wants to just be able to do this: https://youtu.be/CiG8B2T0gys And if you just do it perfectly, you can add a separate tick in the settings so that you can switch it depending on people's tastes.:wink:

**Comment by aarcangeli** [comment]
I released it in version 1.04.0

**Comment by Thanadrax** [comment]
Thank you A LOT. Now this game plays like a charm. Loving your port!
